include/sway/input/cursor.h

```diff
@@ -29,8 +29,8 @@ struct sway_cursor {
 
 void sway_cursor_destroy(struct sway_cursor *cursor);
 struct sway_cursor *sway_cursor_create(struct sway_seat *seat);
-void cursor_send_pointer_motion(struct sway_cursor *cursor, uint32_t time_msec,
-		bool allow_refocusing);
+void cursor_send_pointer_motion(struct sway_cursor *cursor,
```

Comment:
Hmmm. I'd rather keep the last position of the mouse in a field in `sway_cursor`.